### PR TITLE
Updated the database URI in db files & added AI documentation

### LINKED ISSUE #90: Remove the hard-coded database username and password from db files
This makes me realize we currently have the user and pass hard-coded into a const right now. Should definitely change that 😅

_Originally posted by @blake-gilmore in https://github.com/UNLV-CS472-672/2024-S-GROUP4-SMTK/pull/89#discussion_r1532811576_
            
### Description
Right now the files in the db directory:
- username.js
- registerToDB.js
- handleLogin.js

All include the full database authentication information in the URI. This is an obvious security concern and they should all be removed. It's possible other files have added this URI in newer PRs.

### Expected Behavior
No mention of the actual MongoDB username and password for our database should exist in our codebase.

### Possible Solution
Search our files for explicit URI variables that include the username and password, and replace them to match the format in PR #89  for the small-talk/pages/api/auth2.js URI.

### PR DESCRIPTION
**Pull request by salazw1**
## Issue Number
Fixes #90 

## Description

- Changed the URI (masked important information) in username.js, registerToDB.js, and handleLogin.js,
- Added AI utilization documentation in the test file for /accountCreate/page.js

## Testing
No testing needed, files already have coverage.

## Possible Errors
No possible errors.

### CODE REVIEW

**Review by jusasin — APPROVED** [review]
Good documentation on AI and good work on going through the files and changing the different URIs to what they are supposed to be. We still need to fix the URL part of the URI.

**Review by blake-gilmore — APPROVED** [review]
Much much better now. It was suggested before to have the entire request URI in the environment but I think this is more than fine for now.

**Comment by salazw1** [comment]
> Much much better now. It was suggested before to have the entire request URI in the environment but I think this is more than fine for now.

Ohh I see, I can change this real quick! However would this mean that everyone would need to update their own .env.local file correct?

File: small-talk/app/accountCreate/page.test.js [hunk 1]
```diff
@@ -311,6 +311,12 @@ describe('RegisterPage Component', () => {
     });
   });
 
+  /* GENERATIVE AI TOOL USED FOR THIS TEST
```
Comment (aiharc1) [inline small-talk/app/accountCreate/page.test.js:314]: This is a very clear explanation as to which lines used AI. I like this a lot.

**Review by aiharc1 — APPROVED** [review]
Nice work Warren. The changes look good and it worked well on my end. Good job on the AI usage comment

**Comment by aiharc1** [comment]
I will be merging. This has three PR reviews

**Pull request merged**